crypto.css

```diff
@@ -109,5 +105,5 @@ a:hover {
 /* Previews */
 
 #chat .toggle-text {
```

Comment:
Is this rule necessary to keep?

Reply:
Only if we keep the override L88. What do you think? Happy to remove both.

Reply:
I don't mind either, you decide.

Reply:
Heh, whoever takes over the ownership of this repo can improve it, I'll just fix bugs I introduce lol.